operator/pkg/tpath/tree_test.go

```diff
@@ -15,11 +15,11 @@
 package tpath
 
 import (
-	"testing"
-
 	"github.com/ghodss/yaml"
 
 	"istio.io/istio/operator/pkg/util"
+
+	"testing"
```

Comment:
Thanks @adiprerepa . I believe you missed this spot.

Reply:
oops - thanks.

Reply:
should be good now @tariq1890